src/Forms/RemoveTextFromHearImpaired.cs

```diff
@@ -165,7 +165,7 @@ public int RemoveTextFromHearImpaired()
                 if (item.Checked)
                 {
                     Paragraph p = (Paragraph)item.Tag;
-                    string newText = columnHeaderAfter.Text;
+                    string newText = item.SubItems[3].Text.Replace(Configuration.Settings.General.ListViewLineSeparatorString, Environment.NewLine);

```

Comment:
Very close :)
We're not a 100% sure what the ListViewLineSeparator is... could be space...